packages/bug-detectors/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+	"extends": "../../tsconfig.json",
+	"compilerOptions": {
+		"rootDir": ".",
+		"outDir": "dist"
+	},
+	"references": [
+		{
+			"path": "../instrumentor"
```

Comment:
Is this module used? `hooking` is missing.

Reply:
Updated